### PR TITLE
Revert "equip sounds"

### PR DESCRIPTION
**Pull request by actioninja**
Reverts tgstation/tgstation#46387

This was intended to be a low effort immulsion booster but it doesn't work as intended and whatever would be needed to make it work as intended in the equip stack is more work than I'm willing to do right now for such a trivial feature.

Also updates a completely incorrect code comment

:cl:
Remove: Removed equip sounds due to some technical issues, they'll return eventually.
\:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
these were both professional grade sounds and will not be forgotten

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -416,16 +412,14 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 // user is mob that equipped it
 // slot uses the slot_X defines found in setup.dm
 // for items that can be placed in multiple slots
-// note this isn't called during the initial dressing of a player
+// note this ***IS*** called during the initial dressing of a player
```
Comment (spookydonut) [inline code/game/objects/items.dm]: based

**Comment by Fhaxaris** [comment]
iMproVe dON't rEmoVe

**Comment by Qustinnus** [comment]
@actioninja is the only issue roundstart zippers? I can fix it for you if you want if thats the only issue.

**Pull request closed without merging**